lib/import_govdelivery_csv.rb

```diff
@@ -104,7 +104,7 @@ def import_subscriptions
 
         next unless subscribable
 
-        frequency = digest_frequencies.fetch(subscriber.address, Frequency::IMMEDIATELY)
+        frequency = digest_frequencies.fetch(subscriber.address, Frequency::DAILY)
```

Comment:
Seems reasonable, might cause surprises so should probably flag this with Martin if not already done so.